docs/contents/cloud-asset.json

```diff
@@ -0,0 +1,81 @@
+{
+    "title": "Asset",
+    "defaultService": "asset\/readme",
+    "services": [
+        {
+            "title": "Overview",
+            "type": "asset\/readme"
+        },
+        {
+            "title": "AssetService",
```

Comment:
this item and its children should be omitted.

Reply:
Done